Bottom: 5c02105915dc58eeb1becdcb0a768150de6e04e4
Top:    bfb0a601ad8ad3a2ddfeb0cd17db9325b221be08
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-27 14:38:05 -0500

Configure automatic backups


---
diff --git a/roles/nexus3/tasks/main.yml b/roles/nexus3/tasks/main.yml
index 59befd09d..df168c49b 100644
--- a/roles/nexus3/tasks/main.yml
+++ b/roles/nexus3/tasks/main.yml
@@ -259,7 +259,7 @@
     args:
       name: db and blobstores backup
       typeId: script
-      cron: "{{ nexus_backup_cron }}"
+      manual: true
       taskProperties:
         language: groovy
         source: "{{ lookup('template', './templates/backup.groovy.j2') }}"
